src/nvim/eval.c

```diff
@@ -7978,8 +8014,6 @@ static void f_dictwatcherdel(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   }
 
   if (argvars[0].v_type != VAR_DICT) {
-    emsgf(_(e_invarg2), "dict");
-    return;
```

Comment:
why remove this? `f_dictwatcherdel` should not be related to signs stuff, Vim does not have `f_dictwatcherdel`.

Reply:
I really don't know how this slipped in, sorry! Back again now.